sdk/communication/azure-communication-rooms/azure/communication/rooms/_models/_models.py

```diff
@@ -0,0 +1,210 @@
+    """The response object from rooms service.
+
+    :ivar id: Unique identifier of a room. This id is server generated.
+    :vartype id: str
+    :ivar created_date_time: The timestamp when the room was created at the server. The timestamp
+     is in RFC3339 format: ``yyyy-MM-ddTHH:mm:ssZ``.
+    :vartype created_date_time: ~datetime
+    :ivar valid_from: The timestamp from when the room is open for joining. The timestamp is in
+     RFC3339 format: ``yyyy-MM-ddTHH:mm:ssZ``.
+    :vartype valid_from: ~datetime
+    :ivar valid_until: The timestamp from when the room can no longer be joined. The timestamp is
+     in RFC3339 format: ``yyyy-MM-ddTHH:mm:ssZ``.
+    :vartype valid_until: ~datetime
+    :ivar participants: Collection of room participants.
+    :vartype participants: list[~azure.communication.rooms.RoomParticipant]
+    """
+
+    _attribute_map = {
+        'id': {'key': 'id', 'type': 'str'},
+        'created_date_time': {'key': 'createdDateTime', 'type': 'iso-8601'},
+        'valid_from': {'key': 'validFrom', 'type': 'iso-8601'},
+        'valid_until': {'key': 'validUntil', 'type': 'iso-8601'},
+        'participants': {'key': 'participants', 'type': '[RoomParticipant]'},
+    }
+
+    def __init__(
+        self,
+        *,
+        id=None, # type: Optional[str] pylint: disable=redefined-builtin
+        created_date_time=None, # type: Optional[datetime]
+        valid_from=None, # type: Optional[datetime]
+        valid_until=None, # type: Optional[datetime]
+        participants=None, # type: Optional[List[RoomParticipant]]
+        **kwargs
+    ):
+        """
+        :keyword id: Unique identifier of a room. This id is server generated.
+        :paramtype id: str
+        :keyword created_date_time: The timestamp when the room was created at the server. The
+         timestamp is in RFC3339 format: ``yyyy-MM-ddTHH:mm:ssZ``.
+        :paramtype created_date_time: ~datetime
+        :keyword valid_from: The timestamp from when the room is open for joining. The timestamp is in
+         RFC3339 format: ``yyyy-MM-ddTHH:mm:ssZ``.
+        :paramtype valid_from: ~datetime
+        :keyword valid_until: The timestamp from when the room can no longer be joined. The timestamp
+         is in RFC3339 format: ``yyyy-MM-ddTHH:mm:ssZ``.
+        :paramtype valid_until: ~datetime
+        :keyword participants: Collection of room participants.
+        :paramtype participants: list[~azure.communication.rooms.RoomParticipant]
+        """
+        super(RoomModel, self).__init__(**kwargs)
+        self.id = id
+        self.created_date_time = created_date_time
+        self.valid_from = valid_from
+        self.valid_until = valid_until
+        self.participants = participants
+
+    @classmethod
+    def from_room_response(cls, get_room_response,  # type: RoomModel
+            **kwargs  # type: Any
+    ):
+        # type: (...) -> RoomModel
+        """Create CommunicationRoom from a RoomModel.
```

Comment:
CommunicationRoom  has been renamed. Fix this comment accordingly.

Reply:
fixed with the new revision